mesonbuild/coredata.py

```diff
@@ -549,6 +566,7 @@ def parse_cmd_line_options(args):
     'stdsplit':        [UserBooleanOption, 'Split stdout and stderr in test logs.', True],
     'errorlogs':       [UserBooleanOption, "Whether to print the logs from failing tests.", True],
     'install_umask':   [UserUmaskOption, 'Default umask to apply on permissions of installed files.', '022'],
+    'feature_options': [UserFeatureOption, "Override value of all 'auto' features.", 'auto'],
```

Comment:
I initially named it 'automatic_features', @nirbheek suggested 'feature_options', but I agree it would be nice to have 'auto' in the name somehow since it really only apply to auto features. I think `auto_features_override` is what describe it the best but it's a bit long. Maybe just `auto_features`.

e.g. `meson -Dauto_features=disabled` that makes sense, it reads as "automatic features are disabled".

Naming things is really the hard part here... maybe we pay too much attention to small details we'll get used to anyway.

Reply:
I like `-Dauto_features=disabled`

Reply:
Ok, changed to that, also added a bit more doc and release snippet.